Odata-docs/support/support-policy.md

```diff
@@ -0,0 +1,127 @@
+
+Throughout this document, the use of the term *release* is tied to the major version. When we say that we support this release, it means we can ship changes to this package under the same major version, but potentially different minor or patch version. That is to say, if version 7 of a library is actively supported, and the latest version is 7.2.1, we can ship new version such as 7.2.2 or 7.3.0. If it is out of support, it means we'll not release a new 7.x version.
+
+Major version updates (for example from Microsoft.OData.Core 7 to Microsoft.OData.Core 8) often have breaking changes. Thorough testing is advised when updating across major versions. Use the relevant migration guides for guidance on dealing with breaking changes.
+
+Minor version updates do not typically contain breaking changes. However, thorough testing is still advised since new features can introduce regressions. When regressions are detected, they may be rolled back in a subsequent minor or patch release. In the event of an accidental breaking change, we will deprecate the package version, specify an alternate version, and unlist the package version from NuGet.
+
+## Support phases
+
+A major release will go through the following support phases:
+
+- **Preview mode**: Before a new major version is officially release, pre-release versions are published to public to allow people to assess the product and share feedback or point out flaws. Pre-release software may contain critical bugs. Additional we can make breaking changes during the preview phase of a release. As a result, preview releases are not recommended in production. This phase includes preview releases, alpha releases, beta releases and release candidates.
+- **Active support**: The release is in active development, receives security patches, bug fixes and feature updates.
+- **Maintenance mode**: The release will receive security patches and critical bug fixes only . A release will be in maintenance mode at least 6 months before entering End of Life.
+- **End of Life**: The release is out of support. While it may be available for download, it will not receive any new updates, including security patches. Microsoft will provide a minimum of 12 months notification before an OData library goes out of support.
+
+## OData.NET libraries
+
+OData.NET libraries in this context refer to `Microsoft.OData.Core`, `Microsoft.OData.Edm`, `Microsoft.Spatial` and `Microsoft.OData.Client`.
+
+| Version     | Original release date | Latest patch version  | Latest patch release date | Support phase | End of support |
+| ------------|-----------------------|-----------------------|---------------------------|---------------|----------------|
+| 8.x | TBD | 8.0.0-preview-3 | 2024-05-31 | Preview | TBD |
```

Comment:
Should this table be updated?

Reply:
I'm wondering whether we should keep this these dates in the table.

Reply:
I've updated the RC release date though.